### PR TITLE
[RND201-202] Exclude temporary files

### PR DESCRIPTION
**Pull request by Anna-Majda-Mobica**

### CODE REVIEW

File: tests/tests/optionsTests.robot [hunk 1]
```diff
@@ -14,11 +14,14 @@ Test Setup          Prepare For Test
 Test Teardown       Clean After Test
```
Comment (krzysztof-rodzaj-mobica) [inline tests/tests/optionsTests.robot:14]: It does appear that these tests are not using __init__.robot file to prepare environment. I think you should merge changes from "tests" and delete these lines (unless these tests are using something different to prepare or clean)

File: tests/tests/optionsTests.robot [hunk 2]
```diff
@@ -71,15 +71,42 @@ Find and open Extraction language filter
...
+    Wait Until Element Is Visible    ${ReloadBadge}
+
+Exclude temporary files functionality test
+    [Documentation]    Exclude temporary files functionality removes tmp folder from loaded file
+    [Tags]    options
+
+    ${downloaded_file_path}    Set Variable    ${DOWNLOAD_PATH}${test_setup}[archive_name].zip
 
+    Click Add Files Button
+    Ubuntu Upload Test File    ${test_setup}[path]
```
Comment (krzysztof-rodzaj-mobica) [inline tests/tests/optionsTests.robot:96]: changed in "tests" to "Upload Test File"

File: tests/tests/optionsTests.robot [hunk 3]
```diff
@@ -71,15 +71,42 @@ Find and open Extraction language filter
...
+    Should Be Equal As Integers    ${filesCount}    2
+    Click Element    ${FilesList}
+    Wait Until Element Is Visible    ${TemporaryFilesFolder}
+    Click Element    ${OptionsButton}
+    Click Element    ${ExcludeTemporaryFilesSwitch}
+    Click Load Button
+    Click Extract And Save Button    ${extraction_timeout}
+    Wait Until Created    ${downloaded_file_path}
+    Validate and Unzip Test File    ${downloaded_file_path}
+    ${filesCount}    Count Items In Directory    ${DOWNLOAD_PATH}${test_setup}MyProgramme(1)
```
Comment (krzysztof-rodzaj-mobica) [inline tests/tests/optionsTests.robot:111]: Is that correct path to directory?

File: tests/tests/optionsTests.robot (hunk 1 above)
Comment (Anna-Majda-Mobica) [reply tests/tests/optionsTests.robot:14]: Done

File: tests/tests/optionsTests.robot (hunk 2 above)
Comment (Anna-Majda-Mobica) [reply tests/tests/optionsTests.robot:96]: Done

**Comment by Anna-Majda-Mobica** [comment]
As per local branch name change it was needed to all a new PR. I close this one https://github.com/Mobica/innoextract-wasm/pull/96

**Pull request closed without merging**